components/list/ListItem.js

```diff
@@ -88,8 +94,10 @@ const factory = (ripple, ListItemLayout, ListItemContent) => {
           onClick={this.handleClick}
           onMouseDown={onMouseDown}
           onTouchStart={onTouchStart}
+          onKeyDown={this.handleEnter}
+          tabIndex={to ? -1 : 0}
         >
-          {to ? <a href={this.props.to}>{content}</a> : content}
+          {to ? <a tabIndex={0} href={this.props.to}>{content}</a> : content}
```

Comment:
by making the `tabIndex` of the listitem -1, we lose all of the alt text or aria attributes for that element. we want tabinex of `<a>` to be -1, and the on enter to navigate to the provided `to` link